CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 Changes
 =======
+
+# 1.0.0 / 01-07-2020
+### Details
+https://github.com/jenkinsci/datadog-plugin/compare/datadog-0.7.1...datadog-1.0.0
+
+### Changes
+[BACKWARDS INCOMPATIBILITY NTOES]
+* Instead of sending `null` as default value for some tags we now send `unknown`
+* Event messages got changed.
+* Node tag is set by default
+
+[BUGFIX] Fix rate submission. See [179](https://github.com/DataDog/jenkins-datadog-plugin/pull/179) #TODO Confirm if the issue fixed in this PR was in the last release or not
```

Comment:
Does this need to be in the changelog? (i.e. was this a bug that went out with the last release, or something that was broken/fixed after the last release?